lib/public.js

```diff
@@ -116,6 +150,10 @@ class Public {
 
         if (ipAddress) { data.ip = ipAddress }
 
+        if (post) {
+          return this.publicRequest({ resource: TRACK, params: { data: data, test: isTest }, post: post, headers: headers});
+        }
+
         return this.publicRequest({ resource: TRACK, params: { data: data, test: isTest }});
     }
 
```

Comment:
Do we want to add support for `post` in `trackOnce`?

Reply:
we do not